warden/src/main/kotlin/AttestationService.kt

```diff
@@ -324,6 +291,35 @@ abstract class AttestationService {
         }
     }
 
+    private fun <T : PublicKey> processAndroidAttestationResult(
+        keyToBeAttested: T,
+        firstTry: AttestationResult.Android
+    ): KeyAttestation<T> =
+        if (CryptoPublicKey.fromJcaPublicKey(keyToBeAttested) == CryptoPublicKey.fromJcaPublicKey(
+                firstTry.attestationCertificate.publicKey
+            )
+        ) KeyAttestation(keyToBeAttested, firstTry)
```

Comment:
would also refactor this, because it's not clear what's compared to what, and the `if` block has no curly braces, but the `else` block does, although it's also only one statement